__tests__/Snapshottest/__snapshots__/UIOutputComparisonTests.tsx.snap

```diff
@@ -41,6 +41,16 @@ Array [
     className="LibraryItemText"
 >
     String
+</div>,
+  <div
+    className="LibraryItemText"
+>
+    Clockwork
+</div>,
+  <div
+    className="LibraryItemText"
+>
+    DynamoText
```

Comment:
@mjkkirschner These dependencies are referenced in the `RawTypeData.json` file and will cause the comparison to fail if they are not included here.  I am not 100% what changed recently that would make this a new requirement, @ramramps do you have any ideas?  As far as I can tell these packages have been in the `RawTypeData.json` file for a while.

Reply:
Everything passes if I include `Clockwork` and `DynamoText` in the snap.  I wonder if changes with any of the element types made this a requirement at some point.  It's hard to tell exactly when these tests broke without pulling old code and running them until everything passes

Reply:
I think they first start failing at #113 where tests were added around Add-ons

Reply:
ah ok..